docs/faq.md

```diff
@@ -363,10 +363,16 @@ If you're building on an arm64 runner, you might notice something strange about
 
 This is fine for simple C extensions, but for more complicated builds on arm64 it becomes an issue.
 
-So, if the cross-compilation is an issue for you, there is an 'experimental' installer available that's built natively for arm64.
+So, if you want to build macOS arm64 wheels on an arm64 runner (e.g., `macos-14`) on Python 3.8, before invoking cibuildwheel, you should install a native arm64 Python 3.8 interpreter on the runner. On GitHub actions you can set this up easily using the `setup-python` action with something like:
 
-To use this installer and perform native CPython 3.8 building, before invoking cibuildwheel, install the universal2 version of Python on your arm64 runner, something like:
+```yaml
+- uses: actions/setup-python@v5
+  with:
+    python-version: 3.8
+  if: runner.os == 'macOS' && runner.arch == 'ARM64'
+```
 
+There is also an 'experimental' installer available that's built natively for arm64 that you can install manually with something like:
```

Comment:
Isn't the workaround documented just below here already? The workaround shown installs the official build, rather than GitHub's recompilation with a higher MACOSX_DEPLOYMENT_TARGET. (Assuming they are recompiled, not sure they are anymore)

Reply:
It's the same experimental installer used on GHA form arm64.
arm64 on GHA always uses installers from python.org

Reply:
Would it speed up cibuildwheel for a user to pre-install these via setup-python, actually? Might be a bit of a micro-op, just thinking.

Reply:
> Would it speed up cibuildwheel for a user to pre-install these via setup-python, actually? Might be a bit of a micro-op, just thinking.

The only diff in speed I could see would be downloading from GitHub vs python.org for non pre-installed pythons. I don't think it would be noticeable given only 3.8 is not preinstalled on macos-14 (and might add another source of potential failure), we can see in the test workflow that for the action sample build, python 3.9+ cibuildwheel install time is 0s. The same goes for Python 3.11+ on x86_64.

Reply:
If they're functionally equivalent then I'm happy to just change this doc to use `setup-python` and remove the manual install steps. It's more standard practice across GHA to use `setup-python` so if it's functionally equivalent or better to use it, it seems like the docs should suggest that rather than `curl`/`sudo`/`sh` steps suggested currently. I'll change the text to remove the old way, but if that's not preferred feel free to close!

Reply:
The old way works on other CI systems, the GHA one is GHA only. I wonder if tabs here would be better? GHA and general.

Reply:
Good call, changed to tabs!